language/sv.json

```
@@ -1,272 +1,523 @@
+    "energy_intensity_district_heating_commercial": "Energiintensitet fjärrvärme kommersiell",
+    "energy_intensity_district_heating_domestic": "Energiintensitet fjärrvärme hus",
+    "energy_intensity_electric_bus": "Energiintensitet elbuss",
+    "energy_intensity_heat_pumps_commercial": "Energiintensitet värmepumpar kommersiell",
+    "energy_intensity_heat_pumps_domestic": "Energiintensitet värmepumpar hus",
+    "energy_intensity_heavy_bev_truck": "Energiintensitet tunga ellastbilar",
+    "energy_intensity_heavy_hydrogen_truck": "Energiintensitet tunga vätgaslastbilar",
+    "energy_intensity_heavy_trucks_biofuel": "Energiintensitet tunga lastbilar biobränsle",
+    "energy_intensity_hydrogen_commercial": "Energiintensitet vätgas kommersiell",
+    "energy_intensity_hydrogen_domestic": "Energiintensitet vätgas hus",
+    "energy_intensity_internet": "Energiintensitet internet",
+    "energy_intensity_light_bev_truck": "Energiintensitet lätta ellastbilar",
+    "energy_intensity_light_hydrogen_truck": "Energiintensitet lätta vätgaslastbilar",
+    "energy_intensity_machinery_biofuel": "Energiintensitet maskineri (biobränsle)",
+    "energy_intensity_machinery_diesel": "Energiintensitet maskineri (diesel)",
+    "energy_intensity_machinery_electricity": "Energiintensitet maskineri (elektricitet)",
+    "energy_intensity_natural_gas_commercial": "Energiintensitet naturgas kommersiell",
+    "energy_intensity_natural_gas_domestic": "Energiintensitet naturgas hus",
+    "energy_intensity_new_commercial_buildings_standard": "Energiintensitet nya kommersiella byggnader (standard)",
+    "energy_intensity_new_domestic_buildings_standard": "Energiintensitet nya hus (standard)",
+    "energy_intensity_personal_bev": "Energiintensitet elbil",
+    "energy_intensity_petrol_ice": "Energiintensitet bensin ICE",
+    "energy_intensity_railway": "Energiintensitet tåg",
+    "energy_intensity_sea_transport": "Energiintensitet sjötransport",
+    "energy_intensity_walking_biking": "Energiintensitet cykling/gång",
+    "energy_intensity_zero_emission_commercial": "Energiintensitet 0-utsläpp kommersiell",
+    "energy_intensity_zero_emission_domestic": "Energiintensitet 0-utsläpp hus",
+    "engine_efficiency_improvement_personal_vehicle": "Motor effektivitetsförbättring (personliga fordon)",
+    "external_meetings": "Externa möten",
+    "heavy_truck_eco_driving_savings_factor_hgv": "Tunga lastbilar eco-körnings besparningsfaktor",
+    "heavy_truck_efficiency_improvement_factor_hgv": "Tunga lastbilar effektivitetsförbättring",
+    "light_truck_eco_driving_savings_factor_lgv": "Lätta lastbilar eco-körnings besparningsfaktor",
+    "light_truck_efficiency_improvement_factor_lgv": "Lätta lastbilar effektivitetsförbättring",
+    "load_factor_bus_commute": "Belastningsfaktor buss pendling",
+    "load_factor_car_average": "Belastningsfaktor bil genomsnitt",
```

```suggestion
    "load_factor_car_average": "Lastfaktor bil, genomsnitt",
```
Det här är egentligen inget bra uttryck, men lite bättre.